polly/utils/pyscop/isl.py

```diff
@@ -204,7 +204,7 @@ def initialize_isl_methods(self):
         if hasattr(self.__class__, "initialized"):
             return
 
-        self.__class__.initalized = True
+        self.__class__.initialized = True
```

Comment:
Code change: Extra review, please.

Reply:
Seems indeed an error that was supposed to avoid double initialization